js/mercado.js

```diff
@@ -473,6 +474,9 @@ module.exports = class mercado extends Exchange {
             request['from'] = request['to'] - (limit * this.parseTimeframe (timeframe));
         }
         const response = await this.v4PublicGetCoinCandle (this.extend (request, params));
+        if (response['candles'] === undefined) {
```

Comment:
Hehehe ) Got you again!

https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#sanitizing-input-with-safe-methods

```JavaScript
const candles = this.safeValue (response, 'candles', []);
return this.parseOHLCVs (candles, market, timeframe, since, limit);
```

Reply:
I can fix this, let me know ) Thx )

Reply:
@kroitor 
Yes, indeed, good one again! For my excuse, it is way to hot in here to develop software :D
I tested it, works for me.

Reply:
@brandsimon thx for your help! )